### DISCUSSION TITLE
fine-tune task, having a conversation based on that new data

### DISCUSSION THREAD

**Discussion by quincy451** [D7EAD/liboai#40] [Q&A] [answered]
I upload a .jsonl file, I start a fine tune task based on that upload and wait for it to complete.   Now I have a new model in the model list that has the date and organization name on it.  I assume that model is the result of my .jsonl file.  If I change my conversation call from using model gpt-3.5-turbo which works but knows nothing about the jsonl data, to the new model list  I get:
liboai::Response::CheckResponse(): This is not a chat model and thus not supported in the v1/chat/completions endpoint. Did you mean to use v1/completions? (E_APIERROR:0x02)

No idea how to tap into the data the jsonl file added.

**Comment by D7EAD** [comment, marked as answer]
Hi.

This is because the ChatCompletions endpoint does not support fine-tuned models, fine-tuned models are not specifically designed for chat-based conversations. In order to combat this, you can use the regular Completions endpoint with your own conversation tracking to emulate those found in ChatCompletions.

**Reply by quincy451** [reply]
I used the code shown here: https://github.com/D7EAD/liboai/blob/main/documentation/completions/examples/generate_completion.cpp 
passing in my created model and I got no errors about not taking this model and got back a reply which reflected my introduced topic which was gunta.  So that works, now to just work on building bigger better jsonl files for models.

**Reply by D7EAD** [reply]
Good to hear. Good luck on your training.
